BlogTemplate.Tests/Pages/DraftsTests.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using BlogTemplate._1.Models;
+using BlogTemplate._1.Pages;
+using BlogTemplate._1.Tests.Fakes;
+using Xunit;
+
+namespace BlogTemplate._1.Tests.Pages
+{
+    public class DraftsTests
+    {
+        [Fact]
+        public void GetDrafts_ShowDraftSummaries()
+        {
+            IFileSystem fakeFileSystem = new FakeFileSystem();
+            BlogDataStore testDataStore = new BlogDataStore(fakeFileSystem);
+
+            Post draftPost = new Post
+            {
+                Slug = "Draft-Post",
+                Title = "Draft Post",
+                Excerpt = "Draft excerpt",
+                IsPublic = false,
+                PubDate = DateTime.UtcNow,
+            };
+            Post publishedPost = new Post
+            {
+                Slug = "Published-Post",
+                Title = "Published Post",
+                Excerpt = "Published excerpt",
+                IsPublic = true,
+                PubDate = DateTime.UtcNow,
+            };
+
+            testDataStore.SavePost(draftPost);
+            testDataStore.SavePost(publishedPost);
+
+            DraftsModel testDraftsModel = new DraftsModel(testDataStore);
+
+            testDraftsModel.OnGet();
+
+            Assert.Equal(1, testDraftsModel.DraftSummaries.Count());
+            Assert.Equal(draftPost.Id, testDraftsModel.DraftSummaries.First().Id);

```

Comment:
Instead of repeating `testDraftsModel.DraftSummaries.First()`, you could save that into a local variable and it makes the tests more readable IMO.  Also, you probably don't need to test every field - Title and Id are probably sufficient (which also means you could make the test shorter by not setting the Slug or Excerpt).